aten/src/ATen/native/quantized/cpu/qnnpack/src/q8conv/8x8-aarch64-neon.S

```diff
@@ -31,17 +50,27 @@ BEGIN_FUNCTION pytorch_q8conv_ukernel_8x8__aarch64_neon
 
     # Load bias0123, bias4567
     LD1 {v8.4s, v9.4s}, [x5], 32
-    MOV x9, 2
+
+    # Load output channel index
+    # Note since this is an offset into a byte pointer
+    # We do not need to multiply with size of pointer type
+    LDR x9, [sp]
+    # Load pointer to per channel zero points array
+    LDR x10, [x8]
+    # Add offset to the base pointer
+    ADD x10, x10, x9
 
     # v10 := vacc1x0123
     MOV v10.16b, v8.16b
 
     # v11 := vacc1x4567
     MOV v11.16b, v9.16b
     # Load b_zero_point
-    LD1R {v25.8b}, [x8], x9
+    LD1 {v25.8b}, [x10]
+    ADD x8, x8, 8
     # Load a_zero_point
-    LD1R {v24.8b}, [x8], x9
+    LD1R {v24.8b}, [x8]
+    ADD x8, x8, 4
```

Comment:
The two can be replaced with `LD1R {v24.8b}, [x8], 4`.

Reply:
Same as above.